plugins/innogrit/innogrit-nvme.c

```diff
@@ -0,0 +1,534 @@
+
```

Comment:
Please add the SPDX identifier, e.g.

`// SPDX-License-Identifier: GPL-2.0-or-later`

and if you like your copyright notice.